### PR TITLE
Node no activities import hack and revised project structure

### PR DESCRIPTION
**Pull request by bergundy**
Update the docs to reflect changes made in https://github.com/temporalio/sdk-node/pull/215 and https://github.com/temporalio/sdk-node/pull/216

### CODE REVIEW

**Comment by bergundy** [comment]
Please **do not merge** this before the node PR.
We should rerun the build after it has been merged to update the snippets.

**Review by sw-yx — APPROVED** [review]
this is excellent, thank you for taking care of docs alongside the PR. leaving to you to merge since it seems like some more changes are due.

**Pull request merged**
